src/login/logind-dbus.c

```diff
@@ -3093,6 +3101,7 @@ int manager_start_scope(
                 return r;
 
         r = sd_bus_call(manager->bus, m, 0, error, &reply);
+        /* XXX: r == -6 here... WHY? */
```

Comment:
consider using bus_message_dump() to have a look if everything is order inside the message